### PR TITLE
Update location.hostname JSDoc per serverless domains

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8692/app_12858_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8692/HighFidelity-Beta-PR12858-e37655ecdcbdaea5c186e823eaebde1e782e2673.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8692/HighFidelity-Beta-PR12858-e37655ecdcbdaea5c186e823eaebde1e782e2673.exe

**Comment by ctrlaltdavid** [comment]
QA not required because no executable code changes.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8961/app_12858_PR_0552a7242a7d066b9077c8ecb88bd2dceff2548d.apk

File: libraries/networking/src/AddressManager.h [hunk 1]
```diff
@@ -34,15 +34,16 @@ const QString GET_PLACE = "/api/v1/places/%1";
  *
  * @namespace location
  * @property {Uuid} domainID - A UUID uniquely identifying the domain you're visiting. Is {@link Uuid|Uuid.NULL} if you're not
- *     connected to the domain.
+ *     connected to the domain or are at a serverless domain.
```
Comment (MiladNazeri) [inline libraries/networking/src/AddressManager.h]: can we change wording of or and are together? 
connected to the domain or in a serverless domain? maybe?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8962/HighFidelity-Beta-PR12858-8ade4f34f4c70fe6fdb00ad82c0ba9deab22617b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8962/HighFidelity-Beta-Interface-PR12858-8ade4f34f4c70fe6fdb00ad82c0ba9deab22617b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8962/HighFidelity-Beta-PR12858-8ade4f34f4c70fe6fdb00ad82c0ba9deab22617b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8962/HighFidelity-Beta-Interface-PR12858-8ade4f34f4c70fe6fdb00ad82c0ba9deab22617b.exe

**Pull request merged**